src/portal-components/DetailsPage/DetailsPage.tsx

```diff
@@ -1,44 +1,82 @@
 import { cloneDeep, Dictionary } from 'lodash'
+import pluralize from 'pluralize'
 import * as React from 'react'
-import { SynapseComponentWithContext } from '../SynapseComponentWithContext'
+import { useState } from 'react'
+import { BarLoader } from 'react-spinners'
 import {
-  SynapseClient,
   SynapseConstants,
+  SynapseQueries,
   Typography,
 } from 'synapse-react-client'
+import IconSvg from 'synapse-react-client/dist/containers/IconSvg'
+import { LockedFacet } from 'synapse-react-client/dist/containers/QueryContext'
+import { SYNAPSE_ENTITY_ID_REGEX } from 'synapse-react-client/dist/utils/functions/RegularExpressions'
 import {
   insertConditionsFromSearchParams,
   parseEntityIdFromSqlStatement,
 } from 'synapse-react-client/dist/utils/functions/sqlFunctions'
+import { useGetEntityHeaders } from 'synapse-react-client/dist/utils/hooks/SynapseAPI/entity/useGetEntityHeaders'
 import {
+  ColumnType,
   QueryBundleRequest,
   QueryResultBundle,
-  ColumnType,
 } from 'synapse-react-client/dist/utils/synapseTypes/'
+import { SynapseComponent } from 'SynapseComponent'
 import { SynapseConfig } from 'types/portal-config'
 import {
   DetailsPageProps,
   ResolveSynId,
   RowSynapseConfig,
 } from 'types/portal-util-types'
-import injectPropsIntoConfig from './injectPropsIntoConfig'
-import { ExternalFileHandleLink } from 'synapse-react-client/dist/containers/ExternalFileHandleLink'
-import { BarLoader } from 'react-spinners'
 import DetailsPageTabs from './DetailsPageTabs'
-import { SynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
-import { useGetEntityHeaders } from 'synapse-react-client/dist/utils/hooks/SynapseAPI/entity/useGetEntityHeaders'
-import { SYNAPSE_ENTITY_ID_REGEX } from 'synapse-react-client/dist/utils/functions/RegularExpressions'
-import { LockedFacet } from 'synapse-react-client/dist/containers/QueryContext'
-import ToggleSynapseObjects from './ToggleSynapseObjects'
+import injectPropsIntoConfig from '../injectPropsIntoConfig'
+import { handleMenuClick, SideNavMenu } from './SideNavMenu'
+import ToggleSynapseObjects from '../ToggleSynapseObjects'
+import { getComponentId, useScrollOnMount } from './utils'
 
-type State = {
-  queryResultBundle: QueryResultBundle | undefined
-  isLoading: boolean
-  hasError: boolean
+const goToExplorePage = () => {
+  /*
+    Below assumes that going from the details page url up one level will work,
+    for the current set of portals this assumption will hold true.
+  */
+  const lastLocation = window.location.href.split('/')
+  const lastPlace = lastLocation.slice(0, lastLocation.length - 1).join('/')
+  window.location.assign(lastPlace)
+}
+
+function HeadlineWithLink(props: { title: string; id: string }) {
```

Comment:
Component that shows a "link" icon on hover to indicate that a section is link-able

Reply:
Nice!